compiler/AST/stmt.cpp

```diff
@@ -928,7 +928,15 @@ CondStmt::codegen() {
   codegenStmt(this);
 
   if ( outfile ) {
-    info->cStatements.push_back("if (" + codegenValue(condExpr).c + ") ");
+    //here it's very possible that we end up with ( ) around condExpr. It didn't
```

Comment:
Please add a sentence explaining the benefit of stripping the extra () if they are present.